jupyterlab_nvdashboard/apps/gpu.py

```diff
@@ -190,30 +257,29 @@ def _get_color(ind):
         x_range=x_range,
         tools=tools,
     )
-    tot_fig.line(source=source, x="time", y="gpu-total",
-                 color="blue", legend="Total-GPU")
-    tot_fig.line(source=source, x="time", y="memory-total",
-                 color="red", legend="Total-Memory")
+    tot_fig.line(
+        source=source, x="time", y="gpu-total", color="blue", legend="Total-GPU"
+    )
+    tot_fig.line(
+        source=source, x="time", y="memory-total", color="red", legend="Total-Memory"
+    )
     tot_fig.legend.location = "top_left"
 
     pci_fig = figure(
-        title="Total PCI Throughput [MB/s]",
+        title="Total PCI Throughput [/s]",
```

Comment:
I feel like the important distinction here is are we in bits or bytes. Given that it's bytes (correct me if I'm wrong) then I think `[B/s]` would be fine. If the dynamic axis is showing `8GB` as the max range people will probably be forging that the units are a different order of magnitude.

Reply:
Okay, `[B/s]` make sense to me